front/lib/api/assistant/conversation.ts

```diff
@@ -2024,26 +2023,6 @@ async function isMessagesLimitReached({
   };
 }
 
-export function normalizeContentFragmentType({
```

Comment:
https://app.datadoghq.eu/logs?query=%22ContentFragment%20of%20type%20%27file_attachment%27%20being%20created%22&agg_m=count&agg_m_source=base&agg_t=count&cols=host%2Cservice&fromUser=true&messageDisplay=inline&refresh_mode=sliding&storage=hot&stream_sort=desc&viz=stream&from_ts=1732811846975&to_ts=1733416646975&live=true

no log over the past week